app/Services/Transactions/Aggregates/Fees/LastFeeAggregate.php

```diff
@@ -42,9 +42,9 @@ public function aggregate(): array
             ->map(fn ($fee) => $fee->toFloat());
 
         return [
-            'minimum' => $fees->min() ?? 0,
+            'minimum' => $fees->min() ?? 0, // @phpstan-ignore-line
```

Comment:
Ignored because `min()` and `max()` supposedly never return 0 (or as is thought by PHPStan), however tests fail when null-coalescing operator is removed.